admin/formatter/class-metabox-formatter.php

```diff
@@ -68,6 +68,7 @@ private function get_defaults() {
 			'contentAnalysisActive'     => $analysis_readability->is_enabled() ? 1 : 0,
 			'keywordAnalysisActive'     => $analysis_seo->is_enabled() ? 1 : 0,
 			'cornerstoneActive'         => WPSEO_Options::get( 'enable_cornerstone_content', false ) ? 1 : 0,
+			'semrushIntegrationActive'  => WPSEO_Options::get( 'semrush_integration_active', true ) ? 1 : 0,
```

Comment:
Minor: Naming is hard 🙂 I see other settings that enable / disable features have their option name starting with `enable`, for example `enable_cornerstone_content`. Has this been considered for consistency? Why `semrush_integration_active` instead of `enable_semrush_integration`?

Reply:
Two reasons:

1. since we have more than one option related to SEMrush integration, `semrush_` as a prefix seems more logical and helpful (e.g. the tokens for authentication are stored in `semrush_tokens`, another option is in the way for the country code etc.)
2. it was named after the two main toggles in the Features tab, `keyword_analysis_active` and `content_analysis_active`. Also note that the option hasn't been created in this PR, but in P3-73 #15618 which has been already merged.